tests/cephadm/test_config.py

```diff
@@ -0,0 +1,60 @@
+import traceback
+
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def set_multiple_public_network(cls):
+    """
+    To set multiple public network to ceph.
+    Args:
+        cls: cephadm instance object
+
+    Returns:
+        False -> Fail, True -> Pass
+    """
+    out, _ = cls.shell(args=["ceph", "config", "get", "mon", "public_network"])
+    ceph_pub_network = out.rstrip()
+    mask = ceph_pub_network.rsplit(sep="/")
```

Comment:
This step might not work if already multiple public networks have been configured, and user wants to add another to the list. If possible please test that scenario as well.

Reply:
yes manasa i handled this situation now we can set one more network to existing multiple network
```
2022-08-22 03:18:43,326 - INFO - cephci.test_config:110 - Running command cephadm -v shell -- ceph config set mon public_network 10.0.208.0/22,10.0.208.0/23,10.0.208.0/24,10.0.208.0/25,10.0.208.0/26 on 10.0.210.235 timeout 600
2022-08-22 03:18:45,076 - INFO - cephci.test_config:110 - Command completed successfully
```